src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="#"
+          class="header__img"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+
+        <nav class="header__nav">
+          <ul class="header__list">
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li
+              data-qa="hover"
+              class="header__item"
+            >
+              <a
+                href="#"
+                class="header__link"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="header__item">
+              <a
+                href="#"
+                class="header__link"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
```

Comment:
Remember about correct indentation between parent and child elements. For example, elements inside the `header` tag should be indented with 2 spaces.